src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -21,6 +21,9 @@
       <File Include="$(CoreCLRBuildIntegrationDir)*" TargetPath="build" />
       <File Include="$(CoreCLRILCompilerDir)netstandard\*" TargetPath="tools/netstandard" />
       <File Include="sdk\Sdk.props" TargetPath="Sdk" />
+      <File Include="$(MSBuildThisFileDirectory)\..\..\..\..\native\libs\System.Globalization.Native\*" TargetPath="shim_source_code/libs/System.Globalization.Native"/>
```

Comment:
I would call it `source_code` or just `src`. I expect we may end up putting more than just the shims there over time.

Reply:
ok, will rename it to `source_code`